recipe_scrapers/_utils.py

```diff
@@ -145,6 +145,7 @@ def normalize_string(string):
         unescaped_string.replace("\xc2\xa0", " ")
         .replace("\xa0", " ")
         .replace("\u200b", "")
+        .replace("\u00C2", "")
```

Comment:
I'm not sure about this; it is a valid unicode character, so I don't think we should remove it entirely from text we normalize.  What's the situation where it's appearing?

Reply:
For some reason both the outputs for primaledgehealth & panelinha's ingredients are returning this character in the string before the ½ character.

```python
-  '1½ colher (sopa) de azeite',
+  '1Â½ colher (sopa) de azeite',
?    +

-  '½ colher (chÃ¡) de pimenta sÃ\xadria',
+  'Â½ colher (chÃ¡) de pimenta sÃ\xadria',
?   +
-  '¼ de xÃ\xadcara (chÃ¡) de nozes picadas',
+  'Â¼ de xÃ\xadcara (chÃ¡) de nozes picadas',
?   +
```

Reply:
Hmm, weird, OK.  Thanks - I'll take a look at those cases in more detail soon.

Reply:
Text encoding problems drive me mad :|

I'd be OK with replacing this character _for those two scrapers_ - I'm reluctant to add the replacement in the shared `_utils.normalize_string` method though, because -- even though it doesn't affect any of our test data -- there could be recipes that legitimately use the `Â` character in some of the code paths where we use that method.

Reply:
Done, should be good to go now!